### PR TITLE
Dataclasses to typed dict3

### PR DESCRIPTION
**Pull request by crsegerie**

### CODE REVIEW

File: test/services/asset_import/test_import_common.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from kili.queries.project import QueriesProject
 from kili.services.asset_import import import_assets
 from kili.services.asset_import.exceptions import MimeTypeError
+from kili.services.asset_import.types import AssetLike
```
Comment (PierreLeveau) [inline test/services/asset_import/test_import_common.py:9]: rm? does not seem to be used.

**Review by PierreLeveau — APPROVED** [review]

**Comment by PierreLeveau** [comment]
Ready, but do not merge before the release

File: kili/authentication.py [hunk 2]
```diff
@@ -44,9 +45,9 @@ def __init__(
         if api_endpoint and "v1/graphql" in api_endpoint:
             # pylint: disable=line-too-long
             message = (
-                "We are migrating the API to enhance our service,"
```
Comment (p-desaintchamas) [inline kili/authentication.py:47]: I think this whole check could be removed (v1/ was deprecated in january 2021 :))

File: kili/graphql_client.py [hunk 3]
```diff
@@ -9,11 +9,11 @@
 import string
 import threading
 import time
+import urllib
 from datetime import datetime
 from enum import Enum
 
 import websocket
-from six.moves import urllib
```
Comment (p-desaintchamas) [inline kili/graphql_client.py:16]: Can we remove six from `requirements.txt` ?

File: kili/cli/helpers.py [hunk 4]
```diff
@@ -4,9 +4,10 @@
 import warnings
 from typing import Any, Dict, List, Optional
 
+from kili.client import Kili
 from kili.graphql_client import GraphQLClientName
 
-from ..client import Kili
```
Comment (p-desaintchamas) [inline kili/cli/helpers.py:9]: Can we rather fix the associated warnings or is it too hard ?

Comment (PierreLeveau) [reply kili/cli/helpers.py:9]: I agree, this one is fixable

**Pull request merged**